scarpe-components/lib/scarpe/components/tiranti.rb

```diff
@@ -154,6 +154,15 @@ def check_element(props)
     end
   end
 
+  def progress_element(props)
```

Comment:
Ah it's great seeing this concept as a tiranti component!

Reply:
Yeah! I think there are a number of really interesting things we could do with Bootstrap. But some of them (e.g. table) aren't really first-class concepts in Shoes and we'd have to figure out how to fit them in.